Sources/LiveViewNative/Coordinators/LiveSessionCoordinator.swift

```diff
@@ -389,6 +389,20 @@ public class LiveSessionCoordinator<R: RootRegistry>: ObservableObject {
                     ))
                 }
             }
+        case .patch:
```

Comment:
This is a non-blocking comment, just an observation.

It seems like there's a lot of nested conditional logic to the redirect function.
Consider flattening the nested conditions by combining them.

I don't know much about Swift but in Elixir this would be:

```elixir
case int do
 1 -> 
   case letter do
     "a" -> "1a"
     "b" -> "1b"
    end
 2 -> 
    case letter do
     "a" -> "2a"
     "b" -> "2b"
    end
end
```

vs 

```elixir
case {int, letter} ->
  {1,"a"} -> "1a"
  {1,"b"} -> "1b"
  {2,"a"} -> "2a"
  {2,"b"} -> "2b"
end
```